ajax/libs/jquery-ui-bootstrap/package.json

```diff
@@ -7,12 +7,23 @@
     "jquery",
     "bootstrap"
   ],
-  "license": {
-    "name": "MIT License",
-    "url": "http://www.opensource.org/licenses/mit-license.php"
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/jquery-ui-bootstrap/jquery-ui-bootstrap.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jquery-ui-bootstrap/jquery-ui-bootstrap.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/custom-theme/jquery-ui*",
+          "third-party/**/*",
+          "assets/**/*"
```

Comment:
So this will do the job?
```js
        "files": [
          "css/custom-theme/jquery-ui*",
          "third-party/**/*gmap3*.+(js|css|map)",
          "third-party/**/*daterangepicker*.+(js|css|map)",
          "third-party/**/+(*enhance*|*fileinput*|*icon*).+(js|css|map|gif)",
          "third-party/**/+(*{G,g}rid*|*ui*).+(js|css|map)",
          "third-party/**/*wijmo*.+(js|css|map)",
          "assets/**/+(*{F,f}ont-{A,a}wesome*).+(js|css|map|eot|svg|woff|ttf|otf)",
          "assets/**/glyphicons-halflings-regular.+(js|css|map|eot|svg|woff|ttf|otf)",
          "assets/**/apple-touch-icon*.png)",
          "assets/**/+(*bootstrap*|*bs*).png)",
          "assets/**/+(*prettify*|*bootstrap*|*holder*|*html5shiv*|*jquery*|*less*|*respond*).+(js|css|map)"
        ]
```